src/main/java/com/videoclub/suriken/api/AuthenticationController.java

```diff
@@ -0,0 +1,68 @@
+package com.videoclub.suriken.api;
+
+import com.videoclub.suriken.security.MyUserDetailsService;
+import com.videoclub.suriken.jwt.AuthenticationRequest;
+import com.videoclub.suriken.jwt.AuthenticationResponse;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.BadCredentialsException;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDate;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+@RequestMapping("/authenticate")
+@RestController
+public class AuthenticationController {
+
+    String key = "videoclubsurikensecurekeyvideoclubsurikensecurekey";
```

Comment:
Ovo treba da bude environment varijabla, koju procitas u kodu

Reply:
https://stackoverflow.com/questions/44803211/read-environment-variable-in-springboot